src/Symfony/Component/Mailer/Bridge/Mailgun/Http/Api/MailgunTransport.php

```diff
@@ -27,15 +27,17 @@
  */
 class MailgunTransport extends AbstractApiTransport
 {
-    private const ENDPOINT = 'https://api.mailgun.net/v3/%domain%/messages';
+    private const ENDPOINT = 'https://api.%region_dot%mailgun.net/v3/%domain%/messages';
 
     private $key;
     private $domain;
+    private $region;
 
-    public function __construct(string $key, string $domain, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
+    public function __construct(string $key, string $domain, string $region = null, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
```

Comment:
swapping arguments is a BC break, which may be allowed because the class is `@experimental`.

But it also makes the tests failling (https://travis-ci.org/symfony/symfony/jobs/544618001). Because tests are implemented in the component `symfony/mailer` and uses the dependency `symfony/mailgun-mailer (dev-master a2dbd97)`. Meaning, once merged in 4.4, code running `symfony/mailer: 4.3` with `symfony/mailgun-mailer: 4.4` (included symfony test suite) will faill too.

Our choices are:
- allowing BC break, and merging this PR in 4.3 (tests will fail until `mailgun-mailer` be merged to 4.4)
- respecting BC, (ignore failed test with deps=hight until `mailgun-mailer` be merged to 4.4)

Reply:
we definitely should not have used the semver operator for dependencies on experimental packages (as we don't follow semver yet for the BC policy for them). That's not the first time we get this kind of issues these days.

Reply:
I'd suggest merging this as a bug fix so that this becomes less a problem and makes this immediately available so that the break doesn't last long.

Reply:
I agree with @nicolas-grekas here, let's merge as a bug fix in 4.3.

Reply:
@jderusse can you please rebase on 4.3, and patch the composer.json of the Mailer component meanwhile, to fix the lowest version of the mailgun bridge?

Reply:
done. I hesitate between `required-dev: ^4.3.2` and `conflict: <4.3.2`